example/ndpiReader.c

```diff
@@ -831,6 +833,15 @@ static void printFlow(u_int16_t id, struct ndpi_flow_info *flow, u_int16_t threa
     if(flow->bittorent_hash[0] != '\0') fprintf(out, "[BT Hash: %s]", flow->bittorent_hash);
     if(flow->dhcp_fingerprint[0] != '\0') fprintf(out, "[DHCP Fingerprint: %s]", flow->dhcp_fingerprint);
 
+    fprintf(out, " n.p.h.r.s: %d ", flow->n_pckt_human_readable_string);
+
+    if( (flow->detected_protocol.app_protocol == 7) && (flow->n_pckt_human_readable_string == 0) ) printf("!WARNING!");
```

Comment:
please use #define with protocol IDs instead of numbers. Example 7 == NDPI_PROTOCOL_HTTP